apps/theme/pageBuilder/styles/base.scss

```diff
@@ -1,6 +1,16 @@
 @use "./mixins/border";
 @use "./mixins/background";
 
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (prefers-reduced-motion: reduce) {
+  html {
+    scroll-behavior: auto;
+  }
+}
+
```

Comment:
It will work without this, it just won't animate the scrolling, but instead it will just jump to the anchor point

Reply:
@SvenAlHamad @Pavel910 
Are there any additional action items for me?
Create a codemod?
Move css to some other place?

Reply:
Since this is not a blocking problem, I would avoid codemods for this, because they're time consuming, and this is not the only change that needs to be made to the user's project. We have a lot more stuff coming from the rest of the team, that will be "breaking change", and we won't be doing code mods for those.

I noted this change down, and we'll simply write upgrade instructions in one big upgrade guide, with all the things that will be released in 5.34.0. So from point of view, there's nothing you need to do here.

Reply:
@Pavel910 Got it.
I am merging the pull.